scripts/datageneration/data_model.py

```diff
@@ -58,7 +58,7 @@ class Area(BaseModel):
 
 
 class Property(BaseModel):
-    name: str
+    key: str
```

Comment:
Name (I assume descriptor) is added here. Don't we need that for regular tags as well? We also need descriptors for the prompt to tell GPT how to call each tag (bundle).